pkg/api/v1/conversion_generated.go

```diff
@@ -26,7 +26,7 @@ import (
 	resource "k8s.io/kubernetes/pkg/api/resource"
 	unversioned "k8s.io/kubernetes/pkg/api/unversioned"
 	apiv1 "k8s.io/kubernetes/pkg/api/v1"
-	batchv1 "k8s.io/kubernetes/pkg/apis/batch/v1"
+	v1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
```

Comment:
@deads2k i'm sure that sounded like english to you, but it didn't enlighten me :)